client/src/components/Address/CreateAddress.js

```diff
@@ -110,28 +110,32 @@ const CreateAddress = () => {
   return (
     <div>
       {message && <h2>{message}</h2>}
+
+      <SimpleNavBar />
       {options && (
-        <AddressForm
-          street={street}
-          setStreet={setStreet}
-          city={city}
-          setCity={setCity}
-          pin_code={pin_code}
-          setPin_code={setPin_code}
-          state={state}
-          setState={setState}
-          country={country}
-          options={options}
-          changeHandler={changeHandler}
-          submitHandler={submitHandler}
-          title={title}
-          name={name}
-          setName={setName}
-          email={email}
-          setEmail={setEmail}
-          phone={phone}
-          setPhone={setPhone}
-        />
+        <div style={{ marginLeft: "10%" }}>
+          <AddressForm
+            street={street}
+            setStreet={setStreet}
+            city={city}
+            setCity={setCity}
+            pin_code={pin_code}
```

Comment:
you seem to have too many props sent to AddressForm, can we simplify that?
maybe put all the data related props in one object maybe?
like, street, pincode, state, city and etc could be passed as one object right?

Reply:
I didnot get it , can u show me some examples so i can try to implement?

Reply:
there are too many props sent to the component right? can we reduce it? too many props don't look good, just asking if we can reduce it?

Reply:
I tried to change,But I couldnot do ,I will check with mani or chaitanya,

Reply:
We'll discuss that over a call